insights/parsers/ls.py

```diff
@@ -429,7 +429,11 @@ class LSlHFiles(CommandParser, dict):
     """
 
     def parse_content(self, content):
+        self.error_lines = []
         for line in content:
+            if "/bin/ls: cannot access" in line and "No such file or directory" in line:
```

Comment:
@wushiqinlou - plase remove the prefix path "/bin/" from the string, as in the future, we'd change all absolute-command-path to command name only. See [RHINENG-17124](https://issues.redhat.com/browse/RHINENG-17124)